app/graphql/types/query_type.rb

```diff
@@ -1,61 +1,20 @@
 module Types
   class QueryType < Types::BaseObject
-    field :budgets, Types::BudgetType.connection_type, "Returns all budgets", null: false
-    field :budget, Types::BudgetType, "Returns budget for ID", null: false do
-      argument :id, ID, required: true, default_value: false
-    end
-
-    field :comments, Types::CommentType.connection_type, "Returns all comments", null: false
-    field :comment, Types::CommentType, "Returns comment for ID", null: false do
-      argument :id, ID, required: true, default_value: false
-    end
-
-    field :debates, Types::DebateType.connection_type, "Returns all debates", null: false
-    field :debate, Types::DebateType, "Returns debate for ID", null: false do
-      argument :id, ID, required: true, default_value: false
-    end
-
-    field :geozones, Types::GeozoneType.connection_type, "Returns all geozones", null: false
-    field :geozone, Types::GeozoneType, "Returns geozone for ID", null: false do
-      argument :id, ID, required: true, default_value: false
-    end
-
-    field :milestones, Types::MilestoneType.connection_type, "Returns all milestones", null: false
-    field :milestone, Types::MilestoneType, "Returns milestone for ID", null: false do
-      argument :id, ID, required: true, default_value: false
-    end
-
-    field :proposals, Types::ProposalType.connection_type, "Returns all proposals", null: false
-    field :proposal, Types::ProposalType, "Returns proposal for ID", null: false do
-      argument :id, ID, required: true, default_value: false
-    end
-
-    field :proposal_notifications,
-          Types::ProposalNotificationType.connection_type,
-          "Returns all proposal notifications",
-          null: false
-
-    field :proposal_notification,
-          Types::ProposalNotificationType,
-          "Returns proposal notification for ID",
-          null: false do
-      argument :id, ID, required: true, default_value: false
-    end
-
-    field :tags, Types::TagType.connection_type, "Returns all tags", null: false
-    field :tag, Types::TagType, "Returns tag for ID", null: false do
-      argument :id, ID, required: true, default_value: false
-    end
-
-    field :users, Types::UserType.connection_type, "Returns all users", null: false
-    field :user, Types::UserType, "Returns user for ID", null: false do
-      argument :id, ID, required: true, default_value: false
-    end
-
-    field :votes, Types::VoteType.connection_type, "Returns all votes", null: false
-    field :vote, Types::VoteType, "Returns vote for ID", null: false do
-      argument :id, ID, required: true, default_value: false
-    end
+    def self.connection_and_id_fields(name, type)
+      connection_field name.to_s.pluralize.to_sym, type, "Returns all #{name.to_s.pluralize}", null: false
+      id_field name, type, "Returns #{name.to_s.capitalize}", null: false
```

Comment:
I guess we're missing the `for ID` we used to have :thinking:. By the way, what's the reason to introduce `capitalize` here? We're in the middle of a sentence, and we don't use `capitalize` in the `connection_field` method :thinking:.

```suggestion
      id_field name, type, "Returns #{name} for ID", null: false
```